resdk/data_upload/annotate_samples.py

```diff
@@ -0,0 +1,85 @@
+"""Annotate samples in a collection."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from resdk.data_upload.samplesheet import FileImporter, Sample
+
+def annotate_samples(collection, samplesheet_name, local_sheet=True,
+                     verbose=True):
+    """Annotate the samples in a collection based on a sample annotation
```

Comment:
First line of docstring should be one-liner. Also add description of all parameters.